Dockerfile

```diff
@@ -6,12 +6,15 @@ WORKDIR /parity-bridges-ui
 
 RUN apk update
 RUN apk add git
+# Install python/pip
+ENV PYTHONUNBUFFERED=1
+RUN apk add --update --no-cache python3 && ln -sf python3 /usr/bin/python
+RUN python3 -m ensurepip
+RUN pip3 install --no-cache --upgrade pip setuptools
 
-# install app dependencies
-COPY package.json .
-COPY yarn.lock .
-RUN yarn
 COPY . .
+RUN yarn
+RUN yarn downloadCustomTypes
```

Comment:
Isn't this part of pre-build or post-install?

Reply:
pre build, it downloads the custom types files prior to build the image.

Reply:
right, so this line seems unneeded then :)